apps/storybook/docs/d_planning/3_2024-van-v1-naar-v2-beschikbaar.stories.mdx

```diff
@@ -365,6 +388,29 @@ gebruiken dit niet meer.
     </tr>
 </table>
 
+### Image
+
+Wordt niet meer voorzien in v2.
+
+<table  style={{width: 100 + '%'}}>
+    <tr>
+        <th>Naam</th>
+        <th>Toestand</th>
+        <th>v1 - Artifact</th>
+        <th>v1 - Gebruik</th>
+        <th>v2 - Artifact</th>
+        <th>v2 - Gebruik</th>
+    </tr>
+    <tr>
+        <td>[Body [legacy]](/docs/elements-image--documentatie)</td>
+        <td>deprecated</td>
+        <td>elements</td>
+        <td><code>{`<image is="vl-image">`}</code></td>
+        <td>-</td>
+        <td>-</td>
+    </tr>
+</table>
+
```

Comment:
conform de form ook de Image [next] tabel voorzien waarin staat dat je de native 'img' hoort te gebruiken; zodat duidelijk is wat dan wel de bedoeling is